src/Filesystem/Replicate.php

```diff
@@ -26,12 +26,12 @@ class Replicate implements AdapterInterface, MetadataSupporter
     /**
      * @var AdapterInterface
      */
-    protected $master;
+    protected $primary;
```

Comment:
This is a BC break for extending classes, it can't be done on 3.x

Reply:
A solution could be to keep the old property, deprecated it, and assign `$primary` to both properties in the constructor. Then, on master, we could remove the property.

Reply:
```suggestion
    protected $master;
    
    /**
     * @var AdapterInterface
     */
    protected $primary;
```